### PR TITLE
Fix BibTeX "twin" at citation relations

### PR DESCRIPTION
**Pull request by koppor**
Follow-ujp to https://github.com/JabRef/jabref/pull/12123

- Add getter+setter for "cites" to BibEntry
- Refactor CitationsRelationsTabViewModel
- Group citation keys together in JabRef_en.properties
- We fixed an issue where the import of an entry cited by an entry caused a change of the citation key of the citing entry.

I think, CHANGELOG.md entry too special, thus not adding.

### Mandatory checks

- [x] I own the copyright of the code submitted and I licence it under the [MIT license](https://github.com/JabRef/jabref/blob/main/LICENSE)
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if change is visible to the user)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/importer/ImportCommand.java [hunk 1]
```diff
@@ -109,7 +109,7 @@ private void importSingleFile(Path file, SortedSet<Importer> importers, FileChoo
         if (importMethod == ImportMethod.AS_NEW) {
             task.onSuccess(parserResult -> {
                     tabContainer.addTab(parserResult.getDatabaseContext(), true);
-                    dialogService.notify(Localization.lang("Imported entries") + ": " + parserResult.getDatabase().getEntries().size());
+                    dialogService.notify(Localization.lang("%0 entry(ies) imported", parserResult.getDatabase().getEntries().size()));
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/importer/ImportCommand.java:112]: This reads weird. We should make this consistent with plural check if > 1

File: src/main/java/org/jabref/model/entry/BibEntry.java [hunk 2]
```diff
@@ -1124,6 +1126,26 @@ public Optional<FieldChange> addFiles(List<LinkedFile> filesToAdd) {
...
+    /**
+     * Checks {@link StandardField#CITES} for a list of citation keys and returns them.
+     * <p>
+     * Empty citation keys are not returned. There is no consistency check made.
+     *
+     * @return List of citation keys; empty list if field is empty or not available.
+     */
+    public SequencedSet<String> getCites() {
+        return this.getField(StandardField.CITES)
+                   .map(content -> Arrays.stream(content.split(",")))
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/model/entry/BibEntry.java:1138]: can't you use flatmap with filter is Present check

File: src/main/java/org/jabref/gui/importer/ImportCommand.java (hunk 1 above)
Comment (koppor) [reply src/main/java/org/jabref/gui/importer/ImportCommand.java:112]: First introduced at https://github.com/JabRef/jabref/pull/10732. Long call w/ @calixtus  We never wrote it down as follow-up.

TL;DR: It’s not just singular and plural

-    zero → “لم نزرع أي شجرة حتى الآن”
-    one → “لقد زرعنا شجرة ١ حتى الآن”
-    two → “لقد زرعنا شجرتين ٢ حتى الآن”
-    few → “لقد زرعنا ٣ شجرات حتى الآن”
-    many → “لقد زرعنا ١١ شجرة حتى الآن”
-    other → “لقد زرعنا ١٠٠ شجرة حتى الآن”

Longer: https://phrase.com/blog/posts/pluralization/

We decided to remove singular/plural completely and translate only one thing.

@HoussemNasri Seeing https://github.com/JabRef/jabref/pull/9243, WDYT?

File: src/main/java/org/jabref/model/entry/BibEntry.java (hunk 2 above)
Comment (koppor) [reply src/main/java/org/jabref/model/entry/BibEntry.java:1138]: Tried, was very red:

image

Alternative would be

```java
        Arrays.stream(this.getField(StandardField.CITES).orElse("").split(","))
              .map(String::trim)
              .filter(key -> !key.isEmpty())
              .collect(Collectors.toCollection(LinkedHashSet::new));
```

but I don't like the `orElse("")`, because this is hacky, isn't it?

File: src/main/java/org/jabref/gui/importer/ImportCommand.java (hunk 1 above)
Comment (calixtus) [reply src/main/java/org/jabref/gui/importer/ImportCommand.java:112]: Should we write an ADR?

**Review by Siedlerchr — APPROVED** [review]

**Review by calixtus — APPROVED** [review]
